src/Microsoft.Framework.PackageManager/Bundle/BundleProject.cs

```diff
@@ -148,16 +148,13 @@ private void EmitNupkg(BundleRoot root)
                 {
                     sourceStream.CopyTo(targetStream);
                 }
-
-                sourceStream.Seek(0, SeekOrigin.Begin);
-                var sha512Bytes = SHA512.Create().ComputeHash(sourceStream);
-                File.WriteAllText(hashFile, Convert.ToBase64String(sha512Bytes));
             }
 
             // Copy content files (e.g. html, js and images) of main project into "root" folder of the exported package
             var rootFolderPath = Path.Combine(TargetPath, "root");
             var rootProjectJson = Path.Combine(rootFolderPath, Runtime.Project.ProjectFileName);
 
+            root.Operations.Delete(rootFolderPath);
```

Comment:
If we are packing a project without lockfile, we don't want leftover lockfile from previous output